### ISSUE TITLE
oc secret generic docker-registry help is wrong

### ISSUE THREAD

**Issue by bparees** [openshift/origin#18833]
the help says:
```
Usage:
  oc create secret docker-registry NAME --docker-username=user
--docker-password=password --docker-email=email [--docker-server=string]
[--from-literal=key1=value1] [--dry-run] [options]
```

but `--from-literal` is not actually supported.  (it would be nice if it was, along with `--from-file`).

Instead i had to use ` oc create secret generic --type kubernetes.io/dockerconfigjson  mysecret3 --from-file=.dockerconfigjson=path/to/myconfig.json`

**Comment by juanvallejo** [comment]
Upstream PR: https://github.com/kubernetes/kubernetes/pull/60806

**Comment by soltysh** [comment]
I've commented on the PR, but I'll leave a comment here as well. I'm fine with having `--from-file`, since it does serve a specific purpose. But `--from-literal` for docker secret does not make sense, given we have explicit flags for docker-related fields we require.

**Comment by juanvallejo** [comment]
Upstream PR [1] has merged. Flag will be added in next rebase.

1. https://github.com/kubernetes/kubernetes/pull/60806

**Comment by soltysh** [comment]
Not quite next rebase. The next will be 1.10, and this merged into 1.11. You might want to cherry-pick that upstream pr.

**Comment by juanvallejo** [comment]
@soltysh 
> Not quite next rebase. The next will be 1.10, and this merged into 1.11. You might want to cherry-pick that upstream pr.

Thanks, picked changes here: https://github.com/openshift/origin/pull/19151

**Comment by soltysh** [comment]
/close
This is picked as part of #20033
